weave/trace_server/migrations/003_feedback.up.sql

```diff
@@ -0,0 +1,52 @@
+
+CREATE TABLE feedback (
+    /*
+    `id`: The unique identifier for the feedback. This is a UUID.
+    */
+    id String,
+
+    /*
+    `project_id`: The project identifier for the ref. This is an internal
+    identifier that matches the project identifier in the W&B API.
+    It is stored for feedback to allow efficient permissions filtering.
+    */
+    project_id String,
+
+    /*
+    `wb_ref`: The ref the feedback is associated with.
+    Note: the wb prefix is to avoid conflict with React's notion of ref.
+    */
+    wb_ref String,
```

Comment:
might this be `weave_ref`? the `wb_` prefixes are used to refer to data stored in W&B, whereas this is a weave-specific concept? up to you.